powershell/resources/psruntime/BuildTime/Cmdlets/NewAssemblyInfo.cs

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+using System;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Rest.ClientRuntime.PowerShell
+{
+    [Cmdlet("New", "AssemblyInfo")]
+    [DoNotExport]
+    public class NewAssemblyInfo : PSCmdlet
+    {
+        private readonly string assemblyInfoPath = Path.Combine("${$project.baseFolder}", "Properties", "AssemblyInfo.cs");
+        private const string assemblyName = "${$project.title}";
```

Comment:
Please use $(module-name) instead of title here.

Reply:
module-name is Az.xxx, title is without Az, I want it to be without 'Az'